src/vs/workbench/api/browser/mainThreadEditors.ts

```diff
@@ -127,7 +127,7 @@ export class MainThreadTextEditors implements MainThreadTextEditorsShape {
 			// preserve pre 1.38 behaviour to not make group active when preserveFocus: true
 			// but make sure to restore the editor to fix https://github.com/microsoft/vscode/issues/79633
 			activation: options.preserveFocus ? EditorActivation.RESTORE : undefined,
-			override: DEFAULT_EDITOR_ASSOCIATION.id
+			override: EditorResolution.EXCLUSIVE_ONLY
```

Comment:
Not really sure if this has other implications or not?

Reply:
In theory yes someone could pass a terminal or settings editor URI and it would throw an error here as it would resolve to something that isn't a text editor. I think these are edge cases and mostly ok. We don't really have knowledge of "Hey this thing registered with the resolver and actually contains a monaco editor". We have such few `EXCLUSIVE` registrations and they're all specialty URIs that I think this is a safe fix without requiring a much larger refactor

Reply:
Yeah, makes sense